File: angular-ui/bpw-wcm/projects/bpw-wcm-service/src/lib/model/WcmRepository.ts
import { WcmWorkspace } from './WcmWorkspace';

export interface WcmRepository {
    name: string;
    workspacesUrl: string;
    metadata: {[key: string]: string[]};
    backupUrl: string;
    restoreUrl: string;
    activeSessionsCount: number;
    workspaces: WcmWorkspace[];
}

File: wcm-bpm-service/modeshape/modeshape-jcr-rest-boot/src/main/java/com/bpwizard/wcm/repo/rest/jcr/model/WcmRepository.java
package com.bpwizard.wcm.repo.rest.jcr.model;

import java.util.Arrays;

public class WcmRepository {
	private String name;
	private WcmWorkspace workspaces[];
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public WcmWorkspace[] getWorkspaces() {
		return workspaces;
	}
	public void setWorkspaces(WcmWorkspace[] workspaces) {
		this.workspaces = workspaces;
	}
	@Override
	public String toString() {
		return "WcmRepository [name=" + name + ", workspaces=" + Arrays.toString(workspaces) + "]";
	}
	
	
}
